allauth/account/views.py

```diff
@@ -572,6 +573,11 @@ def _action_remove(self, request, *args, **kwargs):
                     "account/messages/email_deleted.txt",
                     {"email": email_address.email},
                 )
+                adapter.send_notification_mail(
+                    "account/email/email_removed",
```

Comment:
Consistency:  the already existing terminology uses `email_deleted` -- so I would prefer to use that here as well.

Reply:
Renamed.